datagrepper/docs/index.rst

```diff
@@ -1,38 +1,32 @@
 .. |crarr| unicode:: U+021B5 .. DOWNWARDS ARROW WITH CORNER LEFTWARDS
 
-Prerequisites
--------------
+Pre-requisites
+--------------
+
+datagrepper interacts with a JSON API with a tool called HTTPie_. All examples
```

Comment:
This sentence reads a little strange to me. The reader interacts with datagrepper's API. Additionally, you _can_ use HTTPie, but you can also use plain old cURL.

Reply:
Okay, I see – I read it as a hard requirement. I understand better now. 👍